### ISSUE TITLE
Add volume to node using PV

### ISSUE THREAD

**Issue by alejandrojnm** [civo/kube100#7]
Right now to be able to do PV and PVC when the cluster is created, Longhorn is installed, the idea would be to be able to use the volumes offered by Civo, and although the cluster is destroyed the volume is persistent and can be reused in another cluster, I also know that It is not possible to load a volume to more than one node at a time.

**Comment by andyjeffries** [comment]
Just for later clarity - this isn't asking for Civo Volumes to be directly attached to pods, but in some way attached to the cluster instances for Longhorn to be able to use.

However, this is tricky if we're talking about keeping the volumes after cluster destruction for later re-attaching (regarding management of them, which nodes they should be re-attached to, how Longhorn would know that it's underlying volume is back and contains its data).

**Comment by alejandrojnm** [comment]
Yep the idea is that when you send to create a PV that creates a live volume, and then attach it to one of the nodes, to then be able to use the PVC from the pods, this results in only the pods that are in the node where the volume can use it, the solution proposed by many even Digital Ocean, Google is to use a Pod with nfs-server and create PVC over NFS.

The part of reloading the volume once destroyed the cluster is already a little more difficult thinking well. But at least the data is there I could load it into a normal intance to save them
